settings.php

```diff
@@ -46,6 +46,10 @@
          1000 => 1000,
     );
 
+    $settings->add(new admin_setting_configtext('attendance/rotateqrcodeinterval',
```

Comment:
can you please drop this under the studentscanmark setting as it relates to that functionality - thanks!